v-next/hardhat/package.json

```diff
@@ -78,9 +78,12 @@
     "@ignored/hardhat-vnext-errors": "workspace:^3.0.0-next.2",
     "@ignored/hardhat-vnext-utils": "workspace:^3.0.0-next.2",
     "@ignored/hardhat-vnext-zod-utils": "workspace:^3.0.0-next.2",
+    "@sentry/node": "^5.18.1",
     "chalk": "^5.3.0",
     "debug": "^4.1.1",
     "enquirer": "^2.3.0",
+    "ethereum-cryptography": "1.1.1",
+    "fp-ts": "1.19.3",
```

Comment:
I think we should remove this. We are using it for the `either` result, we should just handle this in native typescript.